File: Hoteles-Reserva-WS/src/java/models/PurchaseInfo.java
package models;

public class PurchaseInfo {
    private int idPurchaseInfo;
    private double cost, discounts, total;
    private String description, dateTransaction;
    private Purchase purchase;

    public int getIdPurchaseInfo() {
        return idPurchaseInfo;
    }

    public void setIdPurchaseInfo(int idPurchaseInfo) {
        this.idPurchaseInfo = idPurchaseInfo;
    }

    public double getCost() {
        return cost;
    }

    public void setCost(double cost) {
        this.cost = cost;
    }

    public double getDiscounts() {
        return discounts;
    }

    public void setDiscounts(double discounts) {
        this.discounts = discounts;
    }

    public double getTotal() {
        return total;
    }

    public void setTotal(double total) {
        this.total = total;
    }

    public String getDescription() {
        return description;
    }

    public void setDescription(String description) {
        this.description = description;
    }

    public String getDateTransaction() {
        return dateTransaction;
    }

    public void setDateTransaction(String dateTransaction) {
        this.dateTransaction = dateTransaction;
    }

    public Purchase getPurchase() {
        return purchase;
    }

    public void setPurchase(Purchase purchase) {
        this.purchase = purchase;
    }
    
    
}

File: Hotel-Reservacion/Hotel-Reservacion/Models/PurchaseInfo.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Web;

namespace Hotel_Reservacion.Models
{
    public class PurchaseInfo
    {
        private double cost, discounts, total;
        private String description;
        private int idPurchase;

        public int IdPurchase
        {
            get
            {
                return idPurchase;
            }
            set
            {
                idPurchase = value;
            }
        }
        public String Description
        {
            get
            {
                return description;
            }
            set
            {
                description = value;
            }
        }
        public double Cost
        {
            get
            {
                return cost;
            }
            set
            {
                cost = value;
            }
        }
        public double Discounts
        {
            get
            {
                return discounts;
            }
            set
            {
                discounts = value;
            }
        }
        public double Total
        {
            get
            {
                return total;
            }
            set
            {
                total = value;
            }
        }
    }
}